packages/cadl-rlc-emitter/src/modelUtils.ts

```diff
@@ -65,6 +65,7 @@ export function getSchemaForType(
     const schema = getSchemaForModel(program, type, usage, needRef) as any;
     if (usage && usage.includes(SchemaContext.Output)) {
       schema.outputTypeName = `${schema.name}Output`;
+      schema.typeName = `${schema.name}`;
```

Comment:
Is it possible we could define returnType for `getSchemaForType` by leveraging the authering file and our integration testing?

Currently all communications are based on `any` type, that would introduce error like `TypeError: Cannot read properties of undefined (reading 'type')` also we have to guess what will return from end user side.